de.metas.business/src/main/java/de/metas/pricing/conditions/service/impl/PricingConditionsRepository.java

```diff
@@ -291,29 +292,32 @@ private static CurrencyId extractCurrencyId(final I_M_DiscountSchemaBreak discou
 	}
 
 	@VisibleForTesting
-	/* package */ Stream<I_M_DiscountSchemaBreak> streamSchemaBreakRecords(final Properties ctx, final Collection<Integer> discountSchemaIds, final String trxName)
+	/* package */ Stream<I_M_DiscountSchemaBreak> streamSchemaBreakRecords(final Collection<PricingConditionsId> discountSchemaIds)
```

Comment:
rename `discountSchemaIds` to `pricingConditionIds`

Reply:
done